packages/neuron-ui/src/widgets/Icons/icon.tsx

```diff
@@ -40,6 +40,10 @@ import { ReactComponent as NervosDAOSvg } from './NervosDAO.svg'
 import { ReactComponent as ExperimentalSvg } from './Experimental.svg'
 import { ReactComponent as ArrowOpenRightSvg } from './ArrowOpenRight.svg'
 import { ReactComponent as MenuExpandSvg } from './MenuExpand.svg'
+import { ReactComponent as ArrowEndSvg } from './ArrowEnd.svg'
+import { ReactComponent as ArrowNextSvg } from './ArrowNext.svg'
+import { ReactComponent as DisabledArrowEndSvg } from './DisabledArrowEnd.svg'
+import { ReactComponent as DisabledArrowNextSvg } from './DisabledArrowNext.svg'
```

Comment:
Style of svg could be set by CSS so only one set of arrows need to be imported

Reply:
<img width="1265" alt="image" src="https://user-images.githubusercontent.com/22511289/210257575-d0ea5146-5afd-41b5-a4e6-2acd654472d3.png">

I've tried to change the color of arrows by the CSS property `fill` before, but failed.

Reply:
The code of the ArrowNext.svg icon is

```svg
<svg xmlns="http://www.w3.org/2000/svg" width="12" height="12" viewBox="0 0 12 12" fill="none">
  <path d="M3.85547 11.0449L9.71982 6.22299L3.85547 1.40107" stroke="#666666" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/>
</svg>
```

Overriding the `stroke` works

![image](https://user-images.githubusercontent.com/7271329/210260854-6b161af7-b050-4e44-96b5-19b9a56ee193.png)